Docs/seo updates (#36083)
hashicorp/terraform: website/docs/internals/json-format.mdx

---

> page_title: JSON output format overview

Reviewer: ```suggestion
page_title: JSON output format
```

description: >-

>   Terraform can print details about state, configuration, and proposed infrastructure plans to the terminal in JSON format. Learn about the JSON representation.

Reviewer: ```suggestion
  Learn how to configure Terraform to print JSON-formatted details about state, configuration, and proposed infrastructure plans. 
```
